src/Page/SubmitStory/View.elm

```diff
@@ -0,0 +1,121 @@
+module Page.SubmitStory.View exposing (..)
+
+import Css exposing (Style, batch, border, displayFlex, flexGrow, flexShrink, flexWrap, height, hidden, int, overflow, pct, property, rem, width, wrap)
+import Html.Styled exposing (Html, div, h2, iframe, img, p, text)
+import Html.Styled.Attributes as Attr exposing (css)
+import I18n.Keys exposing (Key(..))
+import I18n.Translate exposing (Language(..), translate)
+import Message exposing (Msg)
+import Shared exposing (Model)
+import Theme.FluidScale
+import Theme.Global exposing (centerContent, primaryHeader, roundedCornerStyle)
+
+
+view : Model -> Html Msg
+view model =
+    let
+        t : Key -> String
+        t =
+            translate model.language
+    in
+    div [ css [ centerContent ] ]
+        [ primaryHeader [] (t SubmitStoryHeading)
+        , submitFlex model
+        ]
+
+
+submitFlex : Model -> Html Msg
+submitFlex model =
+    let
+        t : Key -> String
+        t =
+            translate model.language
+    in
+    div
+        [ css
+            [ batch
+                [ displayFlex
+                , property "gap" "1rem"
+                , flexWrap wrap
+                ]
+            ]
+        ]
+        [ description (t SubmitStoryP1) (t SubmitStoryP2)
+        , form model
+        , imageGrid imageList
+        ]
+
+
+description : String -> String -> Html Msg
+description p1 p2 =
+    div [ css [ flexChild ] ]
+        [ p [ css [ Theme.FluidScale.fontSizeMedium ] ] [ text p1 ]
+        , p [] [ text p2 ]
+        ]
+
+
+form : Model -> Html Msg
+form model =
+    let
+        t : Key -> String
+        t =
+            translate model.language
+    in
+    div [ css [ batch [ width (pct 100) ], flexChild ] ]
+        [ h2 [] [ text (t SubmitFormHeading) ]
+        , iframe
+            [ Attr.src (t SubmitFormSrc)
+            , Attr.title (t SubmitFormTitle)
+            , css [ border (rem 0), width (pct 100), height (rem 77) ]
+            ]
+            []
+        ]
+
+
+imageList : List ( String, String )
+imageList =
+    [ ( "/images/submitStory/david-clode-eL4ADAsiOR8-unsplash.jpg", "" )
```

Comment:
Even better if you use just the filename and remove the path from here - moving it to where you render them. This would force future images to also live in that place.

Reply:
good idea I'll do that